static/openapi/faas-spec.json

```diff
@@ -0,0 +1,670 @@
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the function."
+          },
+          "404": {
+            "description": "Not found."
+          },
+          "500": {
+            "description": "Internal server error."
+          }
+        }
+      }
+    },
+    "/api/faas/v1/function/{functionName}": {
+      "delete": {
+        "tags": [
+          "Function"
+        ],
+        "summary": "Delete All versions",
+        "description": "Delete All versions for the specified function name.",
+        "operationId": "handle_all_mm_delete_function",
+        "parameters": [
+          {
+            "name": "X-PhotonIQ-CustomerID",
+            "in": "header",
+            "description": "The customer ID",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "functionName",
+            "in": "path",
+            "description": "The function name",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the function."
+          },
+          "404": {
+            "description": "Not found."
+          },
+          "500": {
+            "description": "Internal server error."
+          }
+        }
+      }
+    },
+    "/api/faas/v1/function/{functionName}/{version}": {
+      "delete": {
+        "tags": [
+          "Function"
+        ],
+        "summary": "Delete the function",
+        "description": "Delete the function for the specified function name.",
+        "operationId": "handle_mm_delete_function",
+        "parameters": [
+          {
+            "name": "X-PhotonIQ-CustomerID",
+            "in": "header",
+            "description": "The customer ID",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "functionName",
+            "in": "path",
+            "description": "The function name",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "version",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the function."
+          },
+          "404": {
+            "description": "Not found."
+          },
+          "500": {
+            "description": "Internal server error."
+          }
+        }
+      }
+    },
+    "/api/faas/v1/functions": {
+      "get": {
+        "tags": [
+          "Function"
+        ],
+        "summary": "Retrieve the function names",
```

Comment:
```suggestion
        "summary": "Retrieve all function names",
```

Reply:
Either works well for meaning, but we want to be consistent throughout, so whichever one we usually use. "Get" is simpler.